domain/src/Access/DomainAccessCheck.php

```diff
@@ -52,11 +52,11 @@ public function applies(Route $route) {
    * {@inheritdoc}
    */
   public function checkPath($path) {
-    $allowed_paths = $this->configFactory->get('domain.settings')->get('login_paths');
+    $allowed_paths = $this->configFactory->get('domain.settings')->get('login_paths', '/user/login\r\n/user/password');
```

Comment:
I disagree here for the following reasons:

* We aren't providing update hook in alpha, which would set the default value and prevent this issue, so providing a default is a workaround.
* The default value is only used if the settings key is not saved in active state.
* There is a legitimate use-case where an admin would set these paths to NULL. Doing so would disallow login on inactive domains. In that case we need the empty check.